templates/microcloud/contact-us.html

```diff
@@ -0,0 +1,153 @@
+    <hr class="p-rule">
+    <div class="col">
+      <h2 class="p-text--small-caps">Submit a question</h2>
+    </div>
+    <div class="col">
+      <div class="row">
+        <div class="col-6">
+          <form action="https://ubuntu.com/marketo/submit" method="post" id="mktoForm_5364">
+            <p>Tell us about your MicroCloud use case</p>
+            <ul class="p-list">
+              <li class="p-list__item">
+                <textarea id="Comments_from_lead__c" name="Comments_from_lead__c" rows="4" maxlength="2000"></textarea>
+              </li>
+              <li class="p-list__item">
+                <label for="firstName" class="is-required">First name</label>
+                <input class="p-form-validation__input" required="" id="firstName" name="firstName" maxlength="255" type="text" />
+              </li>
+              <li class="p-list__item">
+                <label for="lastName" class="is-required">Last name</label>
+                <input class="p-form-validation__input" required="" id="lastName" name="lastName" maxlength="255" type="text" />
+              </li>
+              <li class="p-list__item">
+                <label for="email" class="is-required">Email</label>
+                <input class="p-form-validation__input" required="" id="email" name="email" maxlength="255" type="email" pattern="^[^ ]+@[^ ]+\.[a-z]{2,26}$" />
+              </li>
+              <li class="p-list__item">
+                <label for="phone" class="is-required">Phone number</label>
+                <input class="p-form-validation__input" required="" id="phone" name="phone" maxlength="255" type="tel" />
+              </li>
+              <li class="p-list__item">
+                <label class="p-checkbox">
+                  <input type="checkbox" aria-labelledby="canonicalUpdatesOptIn" class="p-checkbox__input" value="yes">
+                  <span class="p-checkbox__label" id="canonicalUpdatesOptIn">I agree to receive information about Canonical's products and services.</span>
+                </label>
+                <p style="padding-left: 2rem;">By submitting this form, I confirm that I have read and agree to Canonical's <a href="https://ubuntu.com/legal/data-privacy">Privacy Policy</a>.</p>
+              </li>
+            </ul>
+            
+            <hr />
+            <button type="submit" class="p-button--positive" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Form', 'eventAction' : 'cloud contact-us', 'eventLabel' : '{{product}}', 'eventValue' : undefined });">Submit form</button>
+
+            <!-- hidden fields -->
+            <input type="hidden" aria-hidden="true" aria-label="hidden field" name="formid" value="4271" />
+            <input type="hidden" aria-hidden="true" aria-label="hidden field" name="returnURL" value="https://microk8s.io/thank-you" />
+            <input type="hidden" aria-hidden="true" aria-label="hidden field" name="Consent_to_Processing__c" value="yes" />
```

Comment:
This is returning to microk8s, not even returning to microcloud. You may want to update it to canonical.com/microcloud?

Reply:
microk8s link wasn't intentional, but I'm not sure that returning to /microcloud is enough since it doesn't let the user know that the form was submitted. Looking through the conversation in the [design ticket](https://warthogs.atlassian.net/browse/WD-6687) it sounds like the modal and this stand-alone form kept getting confused for one another during planning so there is no microcloud thank you page. @juanruitina what would you like to do here?

Reply:
Can we just copy paste the microk8s thank-you page and use that with the correct content? I think that's better than having it redirect to microk8s, because that's a different site, it's quite confusing and the user doesn't have a clear way to go back to canonical.com/microcloud.

Reply:
@carkod I wasn't suggesting that it route to microk8s at all, again, that link was not intentional

Reply:
Existing microk8s thank you page isn't rebranded, but it looks like the design for it is done now. @juanruitina if you or @Sophie-32 could confirm the content for microcloud/thank-you maybe I could just reuse this design? https://www.figma.com/file/0DTIS3uNNUuDAdCKWrk8Uf/23.10---Microk8s.io-(rebranding)?type=design&node-id=0-1&mode=design&t=YYot1l4o2hASfwkp-0

Reply:
We have a design for the thank-you page [here on Figma](https://www.figma.com/file/hzQYJM9Tui4wpmyZC0k0WS/23.10-C.com---MicroCloud-Rebranding?type=design&node-id=4-5861&mode=design&t=Jo0adHpoL2NUoWgp-0), but don't have the copy for the bottom three blocks for now. Please use that leaving those three blocks out.